src/hoc/withRef.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import Ref from '../addons/Ref'
+
+const withRef = Child => class refHOC extends Component {
```

Comment:
Capitalized classes, please.

nit: Could also just be `Ref` without the `HOC` added :)

Reply:
Nice catch :+1: